glustercli/bash_completion/main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"os"
+
+	"github.com/gluster/glusterd2/glustercli/cmd"
+)
+
+func main() {
+	filename := os.Args[1]
+	cmd.RootCmd.GenBashCompletionFile(filename)
```

Comment:
It'll be good to check (and log/print - you can use go's standard log/fmt package) returned error when `GenBashCompletionFile` is called.

For example, running it as `$ build/bash_completion /etc/bash_completion.d/glustercli.sh` fails silently as only root can write to `/etc/bash_completion.d/`

Reply:
Ok, I will add the log